CommunityToolkit.WinUI.UI.Controls/readme.txt

```diff
@@ -1,30 +1,32 @@
-Thanks for installing the Windows Community Toolkit Controls NuGet package!
+Windows Community Toolkit — UWP Controls
```

Comment:
File's been renamed, you can delete `readme.txt` (don't delete `ReadMe.txt`)

Reply:
Thanks for the heads up!